app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx

```diff
@@ -227,113 +205,53 @@ export class LinkIssueModal extends Component {
-          });
-          this.props.showNotification({
-            message: intl.formatMessage(messages.linkIssueFailed),
-            type: NOTIFICATION_TYPES.ERROR,
-          });
-        });
-    };
-    fetchLogs();
-  }
-
-  renderIssueFormHeaderElements = () => {
-    const {
-      intl: { formatMessage },
-      invalid,
-    } = this.props;
-    return (
-      <>
-        <GhostButton
-          onClick={() => this.props.hideModalAction()}
-          disabled={false}
-          transparentBorder
-          transparentBackground
-          appearance="topaz"
-        >
-          {formatMessage(messages.cancel)}
-        </GhostButton>
-        <GhostButton onClick={this.onLink} disabled={invalid} color="''" appearance="topaz">
-          {formatMessage(messages.linkIssue)}
-        </GhostButton>
-      </>
-    );
-  };
-  renderTitle = (collapsedRightSection) => {
-    const {
-      data: { items },
-      intl: { formatMessage },
-    } = this.props;
-    return collapsedRightSection
-      ? formatMessage(messages.linkIssueForTheTest, {
-          launchNumber: items.launchNumber && `#${items.launchNumber}`,
-        })
-      : formatMessage(messages.link);
-  };
-
-  setItems = (newState) => {
-    this.setState(newState);
-  };
-
-  renderRightSection = (collapsedRightSection) => {
-    const { testItems, selectedItems, loading } = this.state;
-    return (
-      <div className={cx('items-list')}>
-        <ItemsList
-          setItems={this.setItems}
-          testItems={testItems}
-          selectedItems={selectedItems}
-          isNarrowView={collapsedRightSection}
-          loading={loading}
-          eventsInfo={this.props.data.eventsInfo}
-        />
-      </div>
-    );
-  };
+  getFooterButtons = () => ({
+    cancelButton: (
+      <GhostButton
+        onClick={this.props.hideModalAction}
+        color="''"
+        appearance="topaz"
+        transparentBackground
+      >
+        {this.props.intl.formatMessage(COMMON_LOCALE_KEYS.CANCEL)}
+      </GhostButton>
+    ),
+    okButton: (
+      <GhostButton
+        onClick={this.onLink}
+        disabled={this.props.invalid}
+        color="''"
+        appearance="topaz"
+      >
+        {this.props.intl.formatMessage(messages.linkIssue)}
+      </GhostButton>
+    ),
+  });
 
   render() {
     const {
       namedBtsIntegrations,
-      data: { eventsInfo = {} },
+      data: { items, eventsInfo = {} },
       change,
+      intl: { formatMessage },
     } = this.props;
     const { pluginName, integrationId } = this.state;
-    const layoutEventsInfo = {
-      openCloseRightSection: eventsInfo.openCloseRightSection,
-    };
 
     return (
       <DarkModalLayout
-        renderHeaderElements={this.renderIssueFormHeaderElements}
-        renderTitle={this.renderTitle}
-        renderRightSection={this.renderRightSection}
-        eventsInfo={layoutEventsInfo}
+        headerTitle={formatMessage(messages.linkIssue)}
+        footer={
+          <Footer
```

Comment:
We'll have different footers for `MakeDecisionModals` and `post/link/unlink`. May be better to create a different component `Footer` in `darkModalLayout` directory that will have basic footer structure and can be reused by all the dark modals to create their own footers?